SnapshotTests/BubbleViewLayoutLandscapeTests.swift

```diff
@@ -0,0 +1,15 @@
+@testable import GliaWidgets
+import SnapshotTesting
+import XCTest
+
+final class BubbleViewLayoutLandscapeTests: SnapshotTestCase {
+    func test_bubble() {
+        let bubble = ViewFactory.mock().makeBubbleView()
+        bubble.frame = .init(origin: .zero, size: .init(width: 50, height: 50))
+        // If shadow will cause failing test locally or on CI, we should disable it.
```

Comment:
In theory, shadows shouldn't cause issues on CI anymore because they also use Apple Silicon.